modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json

```diff
@@ -4906,6 +4906,829 @@
+									"type": "cssVariable",
+									"value": "table-sm-font-weight"
+								}
+							],
+							"name": "tableSmFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "1.25rem",
+							"label": "table-sm-header-cell-min-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-min-height"
+								}
+							],
+							"name": "tableSmHeaderCellMinHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-2)",
+							"label": "table-sm-header-cell-padding-x",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-padding-x"
+								}
+							],
+							"name": "tableSmHeaderCellPaddingX",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-2)",
+							"label": "table-sm-header-cell-padding-y",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-padding-y"
+								}
+							],
+							"name": "tableSmHeaderCellPaddingY",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-font-size)",
+							"label": "table-sm-header-font-size",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-font-size"
+								}
+							],
+							"name": "tableSmHeaderFontSize",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--font-weight-semi-bold)",
+							"label": "table-sm-header-font-weight",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-font-weight"
+								}
+							],
+							"name": "tableSmHeaderFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-sm-header-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-line-height"
+								}
+							],
+							"name": "tableSmHeaderLineHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-sm-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-line-height"
+								}
+							],
+							"name": "tableSmLineHeight",
+							"type": "String"
+						}
+					],
+					"label": "table-sm",
+					"name": "table-sm"
+				},
+				{
+					"frontendTokens": [
+						{
+							"defaultValue": "var(--color-neutral-2)",
+							"editorType": "ColorPicker",
+							"label": "table-spaced-background-color",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-spaced-background-color"
+								}
+							],
+							"name": "tableSpacedBackgroundColor",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-2)",
+							"label": "table-spaced-padding",
```

Comment:
I implemented this padding because spaced background. Otherwise, it stays like this:
![Screenshot_20220413_143030](https://user-images.githubusercontent.com/47724428/163237447-d328e022-db91-44ab-a68e-e78adeaaae7e.png)

Should I implement the background for the entire table section?

Reply:
For an instance like this where we are only trying to change how it is displayed in the style guide, our fix should also be only in the style guide. In this case you could probably manage the same thing by adding a padding utility class to the same div that you've added the background utility class to.

Reply:
fixed